packages/puppeteer-extra-plugin-recaptcha/package.json

```diff
@@ -64,7 +64,7 @@
   },
   "dependencies": {
     "debug": "^4.1.1",
-    "merge-deep": "^3.0.2",
+    "deepmerge": "^4.2.2",
```

Comment:
Is this dependency needed at all for the recaptcha plugin? I don't see any reference to it.

Reply:
It's being used on line 77 to merge the passed opts with the defaults

Reply:
As @jozsi pointed out, this is not being used on the recaptcha plugin, therefore the dependency can most likely be removed.